config/config.go

```diff
@@ -14,10 +17,17 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
+// TechnicalEndpointBodyMaxSize for /configuration/validate endpoint
+const TechnicalEndpointBodyMaxSize = 8 * 1024
+
+// TechnicalEndpointHeaderContentType for /configuration/validate endpoint
+const TechnicalEndpointHeaderContentType = "application/yaml"
+
 // YamlConfig contains configuration fields of config file
 type YamlConfig struct {
 	// Listen interface and port e.g. "0.0.0.0:8000", "127.0.0.1:9090", ":80"
-	Listen string `yaml:"Listen,omitempty" validate:"regexp=^(([0-9]+[.][0-9]+[.][0-9]+[.][0-9]+)?[:][0-9]+)$"`
+	Listen                  string `yaml:"Listen,omitempty" validate:"regexp=^(([0-9]+[.][0-9]+[.][0-9]+[.][0-9]+)?[:][0-9]+)$"`
+	TechnicalEndpointListen string `yaml:"TechnicalEndpointListen,omitempty" validate:"regexp=^(([0-9]+[.][0-9]+[.][0-9]+[.][0-9]+)?[:][0-9]+)$"`
```

Comment:
I suggest to create new PR for update `Listen and TechnicalEndpointListen` validation type/method.